### DISCUSSION TITLE
Can't create, edit, modify prices, or pricing tables. Lost on what to do.

### DISCUSSION THREAD

**Discussion by ckizer** [medusajs/medusa#7575] [Q&A]
I'm totally stuck on getting pricing to work. I have products, regions, admin and a storefront working. Why can't I enter values?

Can't Edit or Create Prices

Even trying the view options, I cannot enter any prices. **Here's a video of the entire process:**
https://github.com/medusajs/medusa/assets/191738/69e88fe3-3d20-46cc-8950-221e8fd99916

Or Link on CLeanshot: https://share.cleanshot.com/tqJTzy9w

![CleanShot 2024-05-31 at 12 38 59@2x](https://github.com/medusajs/medusa/assets/191738/3c9601de-5d83-463f-8344-bc4fa3303aa7)

**Comment by voxsar** [comment]
This is no answer but rather further explanation of your problem for which i am also looking for an answer

You cannot add the price there because that is a product, if you add a option and then a variation for that option, it will show up there with the ability to add the price

Looks like medusajs does not allow a simple product with a simple set price, i have looked at other threads and so now has put answer on this, i am commenting on all of them until i get a clear answer then I will post it as seperate thread so that people can have a easy one place reference for it

But there needs to be confirmation about what i have figured out for my self after 3 days of scrolling through the github discussion forum

**Comment by SnoozeFreddo** [comment]
lmao, can this be real?
